backend/lib/endpoints/auth.js

```diff
@@ -137,7 +137,9 @@ async function routes(app) {
         throw app.httpErrors.unauthorized("Wrong email or password.");
       }
       req.log.error(err, "Error logging in");
-      throw app.httpErrors.internalServerError();
+      throw app.httpErrors.internalServerError(
```

Comment:
Actually, looking closer, would we be able to parse the `err` object to see if the error is indeed due to too many login attempts? I can foresee some other errors being returned that may not necessarily be related to too many logins.

Reply:
that's also true, according to the documentation of Nginx, the server will throw a 503 error, however, that can happen with other reasons too. I don't think the error message will say anything about the rate-limiting though, but rather just "service unavailable".

Reply:
I think Auth0 returns 429 for too many requests: https://auth0.com/docs/policies/rate-limits. Also, I see this error in [Papertrail](https://my.papertrailapp.com/groups/20211842/events?focus=1227132625207083073&q=program%3Afp-backend-feature1287&selected=1227132625207083073): 

![image](https://user-images.githubusercontent.com/4285073/89248886-a6af4780-d5c5-11ea-9441-15b8d3f7e968.png)

Seems like we could either check the status code for `429` or check the error message for `TooManyRequestsError `.

Reply:
Yes, I agree. We should handle 429 status code error separately

Reply:
Awesome! then 429 it is :)